src/background-scripts/background.es6

```diff
@@ -30,24 +30,29 @@ function onMessage(msg) {
 			chrome.runtime.reload();
 			break;
 		case OPEN_URLS_IN_TABS:
-			chrome.tabs.query({
-					active       : true,
-					currentWindow: true
-				}, (tabs) => {
-					if(tabs.length) {
-						let TabsLeft = msg.tUrls.length;
 
-						// Reverse the url order so that we are opening in the correct order
-						for(let url of msg.tUrls.reverse()) {
-							chrome.tabs.create({
-								url   : url,
-								active: (--TabsLeft) == 0,	// Activate the last tab to be opened
-								index: tabs[0].index+1,
-							});
-						}
-					}
-				}
-			);
+            configs.$loaded.then(function() {

```

Comment:
It seems you must be using spaces vs tabs, causes these reviews to show way more changes than there really are.  Can you use tabs w/ tab space of 4?